lib/views/items_details.erb

```diff
@@ -101,6 +101,21 @@
               </div>
             </div>
           </form>
+              <% pretty_error_message_conversion = { 
+                missing_id: 'Please enter a product ID', 
+                missing_name: 'Please enter a product name',
+                missing_price: 'Please enter a price',
+                missing_quantity: 'Please enter a quantity',
+                invalid_price: 'Please enter a valid price',
+                invalid_quantity: 'Please enter a quantity'
+                } %>
+              <p class="text-danger pl-2">
+              <% @errors.each do |error|%>
+                <% next if error == :invalid_quantity && @errors.include?(:missing_quantity) %>
+                <% next if error == :invalid_price && @errors.include?(:missing_price) %>
+                *<%= pretty_error_message_conversion[error] %><br/>
```

Comment:
Could this be written as:
`<ul><%= pretty_error_message_conversion[error] %></ul>`
instead?

That'd cause a bullet pointed list rather than having to pre-face each line with a * and end with a `<br/>`

Reply:
Putting them in a unordered list spaces them out too much, it looks weird.

Reply:
<img width="1273" alt="screen shot 2018-09-19 at 10 50 36" src="https://user-images.githubusercontent.com/38878719/45745940-e4856b80-bbf9-11e8-9fbd-165c4977b251.png">

Reply:
I know this is in an alert, but it does the same outside of an alert too